### ISSUE TITLE
Phpdoc tags are not completed

### ISSUE THREAD

**Issue by przepompownia** [phpactor/phpactor#1506]
I still deal with a legacy project on Yii which uses lots of magic. It causes using phpdoc tags more often than usual. 

From time to time I try to use the full set of phpactor language server capabilities to check if against some onerousness encountered earlied (and sometimes mentioned here). 

After enabling them I found some new issues with docblocks. Let keep them in separate issues. For this let create any docblock and `@property` tag inside.
```php
/**
 * @prop
              ^complete here
*/
class Phpactor
{
```

**Comment by dantleech** [comment]
The only "docblock" support currently is for doctrine annotations:

https://github.com/phpactor/phpactor/blob/4db0bff91ac32d48a13e4f57b2ebf9a0d5ec6589/lib/Completion/Tests/Integration/Bridge/TolerantParser/DoctrineAnnotationCompletorTest.php

this completion would probably be a `DocblockTagCompletor`, similar to the "Doctrine" one but with a fixed set of tags.

**Comment by dantleech** [comment]
Fixed in #1515

**Comment by przepompownia** [comment]
Works for me, thanks :slightly_smiling_face:
